ci/ci/github.py

```diff
@@ -405,14 +421,24 @@ def github_status(self):
                     log.info(f'failed to get the status for batch {b.id} due to error: {err}')
                     raise
                 if s['state'] != 'cancelled':
-                    if min_batch is None or b.id > min_batch.id:
+                    if min_batch is None:
                         min_batch = b
+                    else:
+                        b_key = (b.id, -self.attempt_id_from_batch(b))
+                        min_key = (min_batch.id, -self.attempt_id_from_batch(min_batch))
```

Comment:
Don't you want the attempt and then the batch?  We should always take the latest attempt, right?  I roughly believe they should be monotonic but this seems clearer.

Reply:
yep